packages/electron/src/electron/main/index.ts

```diff
@@ -8,7 +8,10 @@ import { type UpdateCheckResult, autoUpdater } from "electron-updater";
 
 const store = new Store();
 
-autoUpdater.forceDevUpdateConfig = process.env.NODE_ENV === "development";
+autoUpdater.forceDevUpdateConfig = true;
+
+const ONE_HOUR = 60 * 60 * 1000;
+let updateInfo: UpdateCheckResult | null;
```

Comment:
```suggestion
let updateInfo: UpdateCheckResult | null = null;
```

So you don't have to suppress the TypeScript error that you incorrectly suppress below.